src/doc/unstable-book/src/compiler-flags/sanitizer.md

```diff
@@ -865,6 +867,69 @@ WARNING: ThreadSanitizer: data race (pid=10574)
   Location is global 'example::A::h43ac149ddf992709' of size 8 at 0x5632dfe3d030 (example+0x000000bd9030)
 ```
 
+# RealtimeSanitizer
+RealtimeSanitizer detects blocking calls in realtime contexts. If enabled,
+it won't sanitize the whole program, it needs to be enabled explicitly.
+
+Controlling this sanitizer using attributes isn't yet supported in Rust. The
+sanitizer exports the following functions that allow using it. These functions
+are not stable. When the attributes are added to rust, using them over these
+functions is recommended.
+
+```rust
+extern "C" {
+  /// Enters a realtime context. Can be nested.
+  pub fn __rtsan_realtime_enter();
+  /// Exits a realtime context. After calling this the program could still be in a realtime context,
+  /// as they can be nested.
+  pub fn __rtsan_realtime_exit();
+  /// Disables the sanitizer.
+  pub fn __rtsan_disable();
+  /// Enables the sanitizer again.
+  pub fn __rtsan_enable();
```

Comment:
"These two calls allow you to disable the sanitizer for a given scope. Useful for situations where you know some system call has deterministic execution time under the hood"